### ISSUE TITLE
Chat app is not working!

### ISSUE THREAD

**Issue by umarfarukabu** [mitchtabian/Codingwithmitch-Chat#7]
I am really having some hard time to get the chat functionality to word, when i try sending a message it wont go through the socket to the end receiver!, The Web Socket is firing alight but for some reason the chat wont work, have spend a week now trying to hunt the bug in your codes can't seem to find it yet !

**Comment by sikaili99** [comment]
Hey,what errors are you getting?

**Comment by umarfarukabu** [comment]
Well when i try to send a  public  message it wont show in the message log box and also when i trying creating a private message the send button doesn't work. the socket just disconnect. i don't see any error message in my command line console though ! any suggestions ?These problem is actually with the private chat functionality ! You can chat publicly. But with with Private chat it won't work! @mitchtabian

**Comment by umarfarukabu** [comment]
So i just tried reinstalling a different versions of channels to 3.0.3 it throw me this error(  TypeError: object.__init__() takes exactly one argument (the instance to initialize) ) and then disconnected the socket !

**Comment by sikaili99** [comment]
I will try to look into it, over the weekend. I recently modified this code
to suit my project requirements. But I don't remember getting any error.
Just making sure you have used what is in the requirements.txt file

**Comment by umarfarukabu** [comment]
Yes I have installed the requirement.txt and everything is actually working, It's just the private chat that wont work, Which I don't know why! but am going through the codes as well. I will let you know if am able to figure what went wrong, Thank you for your time !

**Comment by umarfarukabu** [comment]
Hey @Mathewsmusukuma  , So i was able to figure the problem out, it was actually  from the channels-redis==3.0.1 and i had to down grade it to a late version channels-redis==2.4.2 and now everything is working perfectly. Thank you once again for your time !
